tests/regression/58-base-mm-tid/21-broken.c

```diff
@@ -0,0 +1,41 @@
+// PARAM: --set ana.activated[+] symb_locks --set ana.activated[+] var_eq --set exp.extraspecials[+] ZSTD_customMalloc  --set ana.base.privatization mutex-meet-tid --set ana.path_sens[+] threadflag
```

Comment:
Why is this test called "broken"? And what does `ZSTD_customMalloc` have to do with it?